cypress/e2e/functional/opal/manualAccountCreation/adultOrYouthOnly/PO-358_contactDetails.feature

```diff
@@ -18,112 +18,130 @@ Feature: PO-358_Contact Details for adult or youth only
-    When I enter incorrect mobile telephone number "<incorrectMobileNumber>"
-    When I enter incorrect home telephone number "<incorrectHomeTelephone>"
-    When I enter incorrect business telephone number "<incorrectBusinessTelephone>"
+  Scenario Outline: :AC7(358)AC8(PO-419)-negative: When user enters telephone numbers in incorrect format then user will be on the Contact details page
+    When I enter mobile telephone number "<incorrectMobileNumber>"
+    When I enter home telephone number "<incorrectHomeTelephone>"
+    When I enter work telephone number "<incorrectWorkTelephone>"
 
     # Due to changes in PO-360
     #Then I click save and return to tasks
-    Then I click the "Return to account details" button
+    Then I click the "<returnPageButton>" button
     Then I see "Contact details" on the page header
 
-    Then I verify the error message
+    Then I see the error message "Enter a home telephone number, like 01632 960 001" at the top of the page
+    Then I see the error message "Enter a work telephone number, like 01632 960 001 or 07700 900 982" at the top of the page
+    Then I see the error message "Enter a mobile telephone number, like 07700 900 982" at the top of the page
     Then I see "Contact details" on the page header
 
     Examples:
-      | incorrectMobileNumber | incorrectHomeTelephone | incorrectBusinessTelephone               |
-      | 12345678902*$         | 123+ 566 987           | 0123 456                        79  8 89 |
+      | incorrectMobileNumber | incorrectHomeTelephone | incorrectWorkTelephone                   | returnPageButton          |
+      | 12345678902*$         | 123+ 566 987           | 0123 456                        79  8 89 | Return to account details |
+      | 12345678902*$         | 123+ 566 987           | 0123 456                        79  8 89 | Add employer details      |
 
-  Scenario Outline: AC8-negative: When user enters incorrect data into all fields and amends the data will be on account details page
-    When I enter incorrect primary email address "<incorrectPrimaryEmail>"
-    When I enter incorrect secondary email address "<incorrectSecondaryEmail>"
-    When I enter incorrect mobile telephone number "<incorrectMobileNumber>"
-    When I enter incorrect home telephone number "<incorrectHomeTelephone>"
-    When I enter incorrect business telephone number "<incorrectBusinessTelephone>"
+  Scenario Outline: AC8(358)AC9(419)-negative: When user enters incorrect data into all fields and amends the data will be on account details page
+    When I enter primary email address "<incorrectPrimaryEmail>"
+    When I enter secondary email address "<incorrectSecondaryEmail>"
+    When I enter mobile telephone number "<incorrectMobileNumber>"
+    When I enter home telephone number "<incorrectHomeTelephone>"
+    When I enter work telephone number "<incorrectWorkTelephone>"
```

Comment:
generic steps can be used

Reply:
If I'm not saying at least in DOC string how it is an incorrect value

Reply:
the scenario name is specifying that the tests are for incorrect data. the value entered determines if the data is incorrect. 
'When I enter "<incorrectWorkTelephone>" into the "Work telephone number" field' can be used instead of a specific step

Reply:
there isn't much difference in our both steps. I cannot use generic step like happy path

Reply:
Why can the generic step not be used, you are only entering a string into a field? These steps will be replaced when the tests are refactored